sandbox/controls/ListItems.qml

```diff
@@ -331,12 +331,11 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
     StatusListItem {
         title: "List Item with Tags"
         icon.isLetterIdenticon: true
-        tags: [
-            StatusListItemTag {
-                title: qsTr("Tag 1")
-                icon.isLetterIdenticon: true
-            }
-        ]
+        tagsModel: 3
+        tagsDelegate: StatusListItemTag {
+            title: "tag"
+            icon.isLetterIdenticon: true
+        }
```

Comment:
If we have a `tagsModel` property, we already know we want to render tag elements. So we might as well put it in the component, but I like it that you made it a `delegate` so one can do something else.

Maybe we should then no longer call it `tag*` because we could also render other elements? Then, `tagsModel` would be called something else as well I believe...

Curious to hear what @noeliaSD and @alexandraB99 think

Reply:
I could call that `bottomComponentModel` ? 
I don t think I had much choice for doing a delegate because off needed customization

Anyway, happy to rename whatever you prefer

Reply:
`bottomComponentModel` or `bottomSlotModel` sounds good to me. Then also `bottomComponentDelegate`/`bottomSlotDelegate`.

Reply:
Yeah! bottom*Model/Delegate souds good to me. It is more generic!